superset/datasource/api.py

```diff
@@ -35,6 +42,77 @@ class DatasourceRestApi(BaseSupersetApi):
     resource_name = "datasource"
     openapi_spec_tag = "Datasources"
 
+    openapi_spec_component_schemas = (GetOrCreateTableSchema,)
+
+    @expose("/table/get_or_create/", methods=["POST"])
```

Comment:
a table here is a dataset, I think this should live on `/api/v1/dataset`, take a look at: https://github.com/apache/superset/blob/master/superset/datasets/commands/create.py#L57